config/schemas/config.yaml

```diff
@@ -1473,6 +1473,84 @@ properties:
             title: Enable Kafka
             type: boolean
         additionalProperties: false
+      extraRoles:
+        title: Configure extra Roles for application developers
+        description: |
+          Configure extra Roles for application developers
+          The Roles are added to all application developer namespaces configured in user.namespaces
+        type: object
+        properties: {}
+        additionalProperties:
+          properties:
+            rules:
+              title: PolicyRules for this Role
+              type: array
+              default: []
+          additionalProperties: false
```

Comment:
Question: What is the reason for adding this in `additionalProperties` instead of `properties`? AFAIK the common thing would be to add all options in `properties`. But I'm no expert on the config schema yet, so I might just be missing something. 
Same question for the other things you added further down.

Reply:
Since what you configure are objects with "variable/custom" names I did this way as it is also how we do it for configuring constraints, see [here](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.42.0/config/schemas/config.yaml#L1336-L1354). But maybe there is a better way to handle it? Maybe @aarnq has an opinion on this?

Reply:
Yes, this is the way to express `map[string]object`, as the key will become the name of the resource.